client/src/test/java/io/pravega/client/stream/impl/ReaderGroupImplTest.java

```diff
@@ -114,6 +114,11 @@ public void resetReadersToStreamMissingStreamCut() {
     @Test
     @SuppressWarnings("unchecked")
     public void resetReadersToStreamCut() {
+        when(state.getConfig()).thenReturn(ReaderGroupConfig.builder().startFromStreamCuts(ImmutableMap.<Stream,
+                StreamCut>builder()
```

Comment:
Please do not wrap lines like this. Treat the generic type as a unit and keep it on the same line. Either move all of it on the next line or keep it on the previous one. 

Below too.

Reply:
I've made sure not to wrap those lines.